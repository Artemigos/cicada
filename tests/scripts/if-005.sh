# NOTE: this script will print two lines into stderr,
# which will not break the tests
x=foo
if echo hi
    if [ $x = foo ]
        echo before printing break
        break
        echo after printing break
    fi
    echo end of outer if
fi
echo bye

# just a comment
break

echo bye bye
