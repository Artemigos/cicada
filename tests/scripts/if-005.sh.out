hi
before printing break
after printing break
end of outer if
bye
bye bye
